task/sast-snyk-check-oci-ta/0.2/MIGRATION.md

```diff
@@ -0,0 +1,4 @@
+# Migration from 0.1 to 0.2
+
+Inherited from sast-snyk-check task,
+Scanned dir `SOURCE_CODE_DIR`=$(workspaces.workspace.path)/source changed to `SOURCE_CODE_DIR`=$(workspaces.workspace.path)
```

Comment:
Can we provide a bit more explanation why this is the case? Something to the tune of "The snyk check now scans both the source code as well as dependencies in the workspace." or similar.